frontend/src/Home.tsx

```diff
@@ -128,14 +273,59 @@ function Home() {
     );
   }
 
-  return (
-    <>
-      <div className="flex flex-col justify-center items-center pt-20">
-        <h1 className="text-5xl font-bold">There are 2 Routes inside you:</h1>
-        <div className=" pt-16">{JSON.stringify(userQueryResult.data)}</div>
-      </div>
-    </>
-  );
+  if (userQueryResult.isSuccess) {
+    const { draftTimetables, privateTimetables, publicTimetables } =
+      userQueryResult.data;
+
+    return (
+      <>
+        <main className="bg-background min-h-screen text-foreground py-20 px-10 md:px-16">
```

Comment:
```suggestion
        <main className="text-foreground py-6 md:py-12 px-10 md:px-16">
```
Since the navbar is now rendering on this page, the spacing here can be reduced, and since the background will anyways be `bg-background`, the `h-screen` stuff is no longer needed.